src/test/scala/org/broadinstitute/dsde/firecloud/integrationtest/StatisticsSpec.scala

```diff
@@ -0,0 +1,110 @@
+package org.broadinstitute.dsde.firecloud.integrationtest
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.integrationtest.ESIntegrationSupport._
+import org.broadinstitute.dsde.firecloud.model.Document
+import org.broadinstitute.dsde.firecloud.model.Metrics.NumSubjects
+import org.broadinstitute.dsde.rawls.model.{AttributeName, AttributeNumber, AttributeString}
+import org.scalatest.{BeforeAndAfterEach, FreeSpec, Matchers}
+
+class StatisticsSpec extends FreeSpec with Matchers with BeforeAndAfterEach with LazyLogging {
+
+  // "library-ns-1" and "library-ns-2" are codified in reference.conf
+  private val ignoredTuples:Seq[(String,Int)] = Seq(
+    ("should-be-ignored", 944),
+    ("some-other-namespace", 777),
+    ("library-ns-1-2", 888888) // namespace value is really close but not exact!
+  )
+
+  private val ns1Tuples:Seq[(String,Int)] = Seq(
+    ("library-ns-1", 101),
+    ("library-ns-1", 102)
+  )
+
+  private val ns2Tuples:Seq[(String,Int)] = Seq(
+    ("library-ns-2", 220),
+    ("library-ns-2", 225)
+  )
+
+  private def documentsFrom(tuples: Seq[(String,Int)]): Seq[Document] = {
+    tuples map {
+      case (namespace, numSubjects) =>
+        Document(s"$namespace::$numSubjects", Map(
+          AttributeName.withDefaultNS("namespace") -> AttributeString(namespace),
+          AttributeName.withLibraryNS("numSubjects") -> AttributeNumber(numSubjects),
+          // just to see if we can confuse the query
```

Comment:
please make it more clear that this refers to all three - and maybe add a few more?  I dunno.